Makefile

```diff
@@ -21,7 +21,9 @@ export GOFLAGS?=-mod=readonly -trimpath
 BUILD_DATE=$(shell if hash gdate 2>/dev/null; then gdate --rfc-3339=seconds | sed 's/ /T/'; else date --rfc-3339=seconds | sed 's/ /T/'; fi)
 GITCOMMIT=$(shell git log -1 --pretty=format:"%H")
 GITTAG=$(shell git describe --tags --always)
+DEFAULT_STABLE=$(shell curl -SsL https://dl.k8s.io/release/stable.txt)
 GOLDFLAGS?=-s -w -extldflags=-zrelro -extldflags=-znow \
+	-X k8c.io/kubeone/pkg/cmd.defaultInitVersion=$(DEFAULT_STABLE) \
```

Comment:
Can we also use this for `kubeone local`?

Reply:
done